server/flask_server.py

```diff
@@ -0,0 +1,44 @@
+import os
+import csv
+import requests
+from flask import Flask, url_for
+from flask import Flask, request
+import re
+
+app = Flask(__name__)
+
+
+# ルーティングを設定
+@app.route('/', methods=["POST", "GET"])
+
+def send():
+
+    request_data = request.get_data()
+    charset = request.mimetype_params.get('charset') or 'UTF-8'
+    request_text = request_data.decode(charset, 'replace')
+
+
+    if len(request_text) != 0:
+
+        data_list = re.split('[ ,]', request_text)
+
+        change_path = "scripts/data.csv"
+
+        with open(change_path, 'w', encoding='utf-8', newline="") as file:
+            writer = csv.writer(file)
+            writer.writerow(data_list)
+            file.close()
```

Comment:
完全に好みの話なので、無視してもらってもいいです。
```suggestion
    request_data = request.get_data()
    charset = request.mimetype_params.get('charset') or 'UTF-8'
    request_text = request_data.decode(charset, 'replace')

    if len(request_text) != 0:
        data_list = re.split('[ ,]', request_text)
        change_path = "scripts/data.csv"
        with open(change_path, 'w', encoding='utf-8', newline="") as file:
            writer = csv.writer(file)
            writer.writerow(data_list)
            file.close()
```